README.md

```diff
@@ -191,21 +196,4 @@ Mbed TLS can be ported to many different architectures, OS's and platforms. Befo
 -   [What external dependencies does Mbed TLS rely on?](https://tls.mbed.org/kb/development/what-external-dependencies-does-mbedtls-rely-on)
 -   [How do I configure Mbed TLS](https://tls.mbed.org/kb/compiling-and-building/how-do-i-configure-mbedtls)
 
-Contributing
```

Comment:
Please do not delete this section! It's ok to refer to CONTRIBUTING.md for details, but there should be a section stating that we welcome contributions and a summary of the prerequisites.

Reply:
OK, I'll reinstate and link to Contributing.md. I want to minimise any duplication though; this was already misaligned with the previous contributing.md.